ledger/sandbox/src/main/scala/com/digitalasset/platform/sandbox/config/DamlPackageContainer.scala

```diff
@@ -61,9 +61,9 @@ case class DamlPackageContainer(files: List[File] = Nil, devAllowed: Boolean = f
 
   lazy val packageIds: Iterable[String] = archives.map(_.getHash)
 
-  def withFile(file: File): DamlPackageContainer = DamlPackageContainer(file :: files)
```

Comment:
not using `copy` here was causing a bug, which got caught by the test-case on `--allow-dev`